### ISSUE TITLE
After encrypting files, Chinese is displayed garbled

### ISSUE THREAD

**Issue by Woyaoshangbe** [PY44N/LuaObfuscatorV2#13]
After executing the lua Out. lua command, the Output Chinese string is garbled

output:
Hello World
æå°ä¸­æ
this is         ä½ å¥½
this is         World

**Comment by PY44N** [comment]
Yeah, it doesn't compile with Unicode character support right now. I added a task for Unicode support to the roadmap.

**Comment by deathkayn** [comment]
> Yeah, it doesn't compile with Unicode character support right now. I added a task for Unicode support to the roadmap.


Does it support Unicode now?

**Comment by PY44N** [comment]
Still no unicode support
